openfecli/tests/clicktypes/test_hyphenchoice.py

```diff
@@ -8,7 +8,7 @@ class TestHyphenAwareChoice:
     ])
     def test_init(self, value):
         ch = HyphenAwareChoice([value])
-        assert ch.choices == ["foo-bar-baz"]
+        assert ch.choices == ("foo-bar-baz",)
```

Comment:
https://click.palletsprojects.com/en/stable/api/#click.Choice click now casts this to a tuple, shouldn't be an issue for us.